static/data/contributors.json

```diff
@@ -3,5 +3,6 @@
     {"name": "Michael Fisher", "ext": "3144", "url": "https://s.gravatar.com/avatar/a5702d4f23a3869de0402ddb53a788c6?s=800"},
     {"name": "Sara Minerva", "ext": "3509", "url": "https://s.gravatar.com/avatar/1a17b8243224a52f34bc73fd654f129e?s=800"},
     {"name": "Sumner Robinson", "ext": "3138", "url": "https://s.gravatar.com/avatar/3380deb20db45e697c40c2444c3b615b?s=800"},
-    {"name": "Brian Longwell", "ext": "3145", "url": "https://media-exp1.licdn.com/dms/image/C4E0BAQHR3nDWRbZROw/company-logo_200_200/0/1614707005483?e=2159024400&v=beta&t=Do7S2L8h93-T6lH_XvRiYdUrYYgugpVQtWAvH9fLFAs"}
+    {"name": "Brian Longwell", "ext": "3145", "url": "https://media-exp1.licdn.com/dms/image/C4E0BAQHR3nDWRbZROw/company-logo_200_200/0/1614707005483?e=2159024400&v=beta&t=Do7S2L8h93-T6lH_XvRiYdUrYYgugpVQtWAvH9fLFAs"},
+    {"name": "Test User", "ext": "3145", "url": "https://media-exp1.licdn.com/dms/image/C4E0BAQHR3nDWRbZROw/company-logo_200_200/0/1614707005483?e=2159024400&v=beta&t=Do7S2L8h93-T6lH_XvRiYdUrYYgugpVQtWAvH9fLFAs"}
```

Comment:
@SMinervaTelware, I would say don't add the test user.  I can delete the entry I made for you so you can do it and get your PR if you want.